src/_playground/_content.scss

```diff
@@ -58,7 +58,7 @@
     }
 
     &__tile-background {
-        background: #F3F4F5;
+        background: #f3f4f5;
```

Comment:
switching to lowercase for HEX looks weird to me.

Reply:
It's easy enough to change.  I suppose it would make sense to keep the stylelint consistent with [fundamental-styles](https://github.com/SAP/fundamental-styles/blob/master/.stylelintrc.json)
They [use upper case there](https://github.com/SAP/fundamental-styles/blob/fac41376f8eba6c4e55f884172c754886f76adb6/docs/_sass/_docs-utils.scss#L4)